lang/python/setup_pip.py

```diff
@@ -323,7 +323,7 @@ def run(self):
                 lambda: build_commands(make_cmds, wt_dir, env), [],
                 'wiredtiger make')
             open(built_sentinal, 'a').close()
-            shutil.move(os.path.join(conf_make_dir, python_rel_dir, '_wiredtiger.so'), '.')
```

Comment:
@alisonfel Why we remove this line? We don't build this anymore?

Reply:
I don't believe we need this line. That change reverts back to relying on the setup process building the shared python API library (as it was doing prior to our changes). Trying to bypass the `build_ext` process by dropping in our own binary seemed to cause more issues and also doesn't stop it from compiling its own python `_wiredtiger` library.

I think we should only need the CMake build to generate the libwiredtiger archive and the SWIG files (which was how it was done in the autoconf build)